docs/cluster-configuration.md

```diff
@@ -32,8 +32,8 @@ From there a user can call `cluster.up()` and `cluster.down()` to create and rem
 In cases where `mcad=False` a yaml file will be created with the individual Ray Cluster, Route/Ingress and Secret included.<br>
 The Ray Cluster and service will be created by KubeRay directly and the other components will be individually created.
 
-## Ray Cluster Configuration in a Kubernetes environment
-To create a Ray Cluster using the CodeFlare SDK in a Kubernetes environment an `ingress_domain` must be passed in the Cluster Configuration.
+## Ray Cluster Configuration in a Vanilla Kubernetes environment (Non-OpenShift)
+To create a Ray Cluster using the CodeFlare SDK in a Vanilla Kubernetes environment an `ingress_domain` must be passed in the Cluster Configuration.
 This is used for the creation of the Ray Dashboard and Client ingresses.
 
 `ingress_options` can be passed to create a custom Ray Dashboard ingress, `ingress_domain` is still a required variable for the Client ingress.
```

Comment:
```suggestion
`ingress_options` can be passed to create a custom Ray Dashboard ingress, `ingress_domain` is still a required variable for the Client route/ingress.
```
Just a small nit that should be added for the sake of making things as clear as possible.
`ingress_domain` is required for OpenShift when `local_interactive` is enabled for now. This may change soon.

Reply:
Good one thanks Mark! I added a temporary note, and also added `local_interactive` field on the docs now.